java/src/jmri/configurexml/StoreAndCompare.java

```diff
@@ -239,7 +239,8 @@ public static boolean checkFile(File inFile1, File inFile2) throws Exception {
                 "<layout-config",   // Linux seems to put attributes in different order
                 "<?xml-stylesheet", // Linux seems to put attributes in different order
                 "    <memory systemName=\"IMCURRENTTIME\"", // time varies - old format
-                "    <modifier>This line ignored</modifier>"
+                "    <modifier>This line ignored</modifier>",
+                "    <memory"       // Ignore memory variable value changes, including to/from null.
```

Comment:
Maybe a space should be added after memory. Currently this line also matches `"    <memoryIcon"`and similar lines.

Reply:
The memory process will need some special logic.  A memory with a null value does not have the " value=" content. 

The cases are "    <memory>" or "    <memory value" but not "    <memoryIcon".